Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM registry.opensource.zalan.do/stups/node:5.10-16
+FROM registry.opensource.zalan.do/stups/node:6.9-cd32
```

Comment:
Nah, because we have parallel pipelines with node. Depending on the order of releasing an image you might get 6.x during one build and 7.x during the other.